README.md

```diff
@@ -364,6 +386,27 @@ to find instruction and links to exact configuration files and final checkpoints
 |MaskRCNN|INT8 (per-tensor for weights)|COCO2017|bbox: 37.27 (0.06)<br/>segm: 33.54 (0.02)|
 |MaskRCNN|Sparsity 50% (Magnitude)|COCO2017|bbox: 36.93 (0.40)<br/>segm: 33.23 (0.33)|
 
+### ONNX models
+
+#### Classification
+
+| ONNX Model |    Compression algorithm    | Dataset  | Accuracy (Drop) % |
+|:----------:|:---------------------------:|:--------:|:-----------------:|
+|resnet50| Post-Traininig Quantization | ImageNet |   74.63 (0.21)    | 
+|shufflenet|            Post-Traininig Quantization           |  ImageNet   |    47.25(0.18)    |
+|googlenet|            Post-Traininig Quantization           |   ImageNet  |    66.36(0.3)     |
+|squeezenet1.0|           Post-Traininig Quantization           |   ImageNet   |    54.3(0.54)     |
+|mobilenetv2|           Post-Traininig Quantization           |   ImageNet  |    71.38(0.49)    |
+|densenet|           Post-Traininig Quantization            |   ImageNet  |    60.16(0.8)     |
+|vgg16|           Post-Traininig Quantization         |   ImageNet  |    72.02(0.0)     |
+
+#### Object Detection
+
+| ONNX Model |    Compression algorithm    | Dataset  | Accuracy (Drop) % |
+|:----------:|:---------------------------:|:--------:|:-----------------:|
+|ssd|            Post-Traininig Quantization           |    MSCOCO  |    20.17(0.17)    |
+|tinyyolov2|            Post-Traininig Quantization            |  VOC2012   |      29.03(0.23)      |
+
```

Comment:
Fix typos and source-level formatting, and use consistent names for models, datasets, compression algorithms and metric names as with the rest of the frameworks.

Reply:
done